From 2693ced4c41f838a4285a0e17928d0a7d5bb2705 Mon Sep 17 00:00:00 2001
From: Ricard Wanderlof <polluxsynth@butoba.net>
Date: Mon, 4 Apr 2022 21:27:19 +0200
Subject: [PATCH 1852/1936] Add likely missing else

Add missing else. This avoids messages to be forwarded being sent
to remote as well.
---
 src/Misc/MiddleWare.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Misc/MiddleWare.cpp b/src/Misc/MiddleWare.cpp
index 89c29649..3cbf25cd 100644
--- a/src/Misc/MiddleWare.cpp
+++ b/src/Misc/MiddleWare.cpp
@@ -2280,7 +2280,7 @@ void MiddleWareImpl::bToUhandle(const char *rtmsg)
         if(forward) {
             forward = false;
             handleMsg(rtmsg, true);
-        } if(broadcast)
+        } else if(broadcast)
             broadcastToRemote(rtmsg);
         else
             sendToCurrentRemote(rtmsg);
-- 
2.47.0

